### PR TITLE
Clarify how os.Verify reports UNAVAILABLE

### PR DESCRIPTION
**Pull request by liulk**

### CODE REVIEW

File: os/os.proto [hunk 1]
```diff
@@ -334,7 +343,7 @@ message StandbyState {
   enum State {
     UNSPECIFIED = 0;
     // The Target does not support dual Supervisors.
-    UNSUPORTED = 1;
+    UNSUPPORTED = 1;
```
Comment (hellt) [inline os/os.proto:346]: given the field name changes, should we bump the version of the service as well?

Comment (liulk) [reply os/os.proto:346]: Done

**Review by dplore — APPROVED** [review]

**Comment by dplore** [comment]
Will hold this open for 2 weeks for feedback until Feb 7, 2023.

**Comment by samribeiro** [comment]
LGTM

**Pull request merged**